client/blocks/inline-help/contextual-help.js

```diff
@@ -766,329 +868,371 @@ const videosForSection = {
 			link: 'https://www.youtube.com/embed/Vyr-g5SEuIA',
-			title: 'Change Your Language Settings',
-			description:
-				'Find out how to change your blog or website language and your interface language settings on WordPress.com.',
+			title: translate( 'Change Your Language Settings' ),
+			description: translate(
+				'Find out how to change your blog or website language and your interface language settings on WordPress.com.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/EUuEuW_LCrc',
-			title: 'Activate Free Email Forwarding',
-			description:
-				'Find out how to activate free email forwarding from an address using a custom domain registered through WordPress.com.',
+			title: translate( 'Activate Free Email Forwarding' ),
+			description: translate(
+				'Find out how to activate free email forwarding from an address using a custom domain registered through WordPress.com.'
+			),
 		},
 	],
 	'post-editor': [
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/hNg1rrkiAjg',
-			title: 'Set a Featured Image for a Post or Page',
-			description:
-				'Find out how to add a featured image where available on your WordPress.com or Jetpack-enabled WordPress site.',
+			title: translate( 'Set a Featured Image for a Post or Page' ),
+			description: translate(
+				'Find out how to add a featured image where available on your WordPress.com or Jetpack-enabled WordPress site.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/dAcEBKXPlyA',
-			title: 'Add a Contact Form to Your Website',
-			description: 'Find out how to add a contact form to your WordPress.com site.',
+			title: translate( 'Add a Contact Form to Your Website' ),
+			description: translate( 'Find out how to add a contact form to your WordPress.com site.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/ssfHW5lwFZg',
-			title: 'Embed a YouTube Video in Your Website',
-			description:
+			title: translate( 'Embed a YouTube Video in Your Website' ),
+			description: translate(
 				'Find out how to embed a YouTube video in your content (including posts, pages, and even comments) ' +
-				'on your WordPress.com or Jetpack-enabled WordPress website or blog.',
+					'on your WordPress.com or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/_tpcHN6ZtKM',
-			title: 'Schedule a Post',
-			description: 'Find out how to schedule a post on your WordPress.com website or blog.',
+			title: translate( 'Schedule a Post' ),
+			description: translate(
+				'Find out how to schedule a post on your WordPress.com website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/V8UToJoSf4Q',
-			title: 'Add a Simple Payment Button',
-			description: 'Find out how to add a payment button to your WordPress.com website.',
+			title: translate( 'Add a Simple Payment Button' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Add a simple payment button.' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/localized-support-urls-inline-contextual-help)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).